apps/desktop/src/components/views/settings-view/claude/claude-section.tsx

```
@@ -15,75 +19,159 @@ export function ClaudeSection() {
+          <p className="text-sm text-muted-foreground">Claude Code CLI installation status</p>
         </div>
+        <button
+          onClick={refreshStatus}
+          disabled={isLoading}
+          className={cn(
+            'p-2 rounded-lg transition-colors',
+            'hover:bg-muted text-muted-foreground hover:text-foreground',
+            'disabled:opacity-50 disabled:cursor-not-allowed'
+          )}
+          title="Refresh status"
+        >
```

Biome (2.3.13)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)